docs/FEDERATION-DOWNLOAD.md

```diff
@@ -0,0 +1,92 @@
+# CWA Federation Key Download Service
+
+This is a spring boot [ApplicationRunner](https://docs.spring.io/spring-boot/docs/current/api/org/springframework/boot/ApplicationRunner.html) service (running as a cronjob). The app will deal with the download, sematic validation, extraction, and storage of the keys from the federation gateway. The download service leverages the download API of the federation gateway and will trigger downloads of available batches since the last time it executed. For the initial release the download service will use the polling mechanism provided by the federation gateway based on `batchTag` and `Date` combinations and it will keep track of its last processed state within the database. When and if the callback service integration is fully realized the polling mechanism would mainly be used for mass loading scenarios, and this service will then only download the persisted individual batches where notifications have been received.
+
+On the download of keys from the federation gateway a process of normalization needs to take place. This is done to enable the keys to be consumable by the DE CWA app as not all countries support the same approach which is required for the CWA app. The means the following:
+
+- For keys from countries which support the daysSinceOnset scenario will need to be converted into an appropriate transmission risk level
+- For keys from countries which support the transmission risk level we might need to translate the value to something which is reasonable for the RKI
+- For keys from countries which support neither, or a key doesn't happen to have either value then there will be a need to provide a reasonable default
```

Comment:
I discussed with @EugenM-SAP regarding this details TRL value. This would really apply as the scenario has now changed since this was originally drafted to cases where a country provides a DSOS value which we do not know how to map to a TRL. This was the agreement with @tklingbeil

Reply:
For now I've adjusted the text